packages/react/.eslintrc.js

```diff
@@ -30,13 +30,13 @@ const primitivePatterns = templateJoin(
     'Q',
     'R',
     'S',
-    // 'T',
-    // 'U',
-    // 'V',
-    // 'W',
-    // 'X',
-    // 'Y',
-    // 'Z'
+    'T',
```

Comment:
If all primitives are done, can we remove these and go back to `src/primitives/**/*`?

Reply:
There is no going back where we've never been 😄 

All jokes aside will update the _.eslintrc.js_ in a follow up

Reply:
Went ahead and updated the lint config to remove the need `templateJoin`